Assets/Scenes/Scripts/DestroyByContact.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DestroyByContact : MonoBehaviour {
+	public GameObject explosion;
```

Comment:
I noticed this, along with all other variables here are public yet not referenced in any other scripts. Is this necessary to be public? If you are using public to assign references in the inspector then use [SerializeField] above the member like this
```suggestion
	[SerializeField]
	private GameObject explosion;
```

Reply:
This applies to all other unecessarily public variables